lib/hobbes/boot/amapping.hob

```diff
@@ -52,7 +52,9 @@ instance (p++s=a, Function f p rf, Function rf s r) => Function f a r where
 class Array xs x | xs -> x where
   size     :: xs -> long
   element  :: (xs, long) -> x
+  elementM  :: (xs, long) -> (()+x)
   elements :: (xs, long, long) -> [x]
+  elementsM :: (xs, long, long) -> [x]
```

Comment:
Why is `elements` in on this?  Isn't it always safe?

Reply:
Isn't that just a bug?  There's nothing about the type that excludes a safe result, unlike `element`.  Why build ceremony/scaffolding around a bug?

Reply:
will be addressed in #373